doc/api/index.rst

```diff
@@ -227,6 +227,7 @@ and store them in the GMT cache folder.
     datasets.load_fractures_compilation
     datasets.load_hotspots
     datasets.load_japan_quakes
+    datasets.load_map_tiles
```

Comment:
<img width="929" alt="image" src="https://user-images.githubusercontent.com/3974108/195764331-79c06330-ae8d-4a5b-b3fd-e610ce3985c0.png">

Looking at the table. I think it's better to put the deprecated load functions in a separate table. Of course, this can be done in a separate PR.

Reply:
Done in #2228.